src/dstack/_internal/server/services/runner/client.py

```diff
@@ -17,6 +17,7 @@
 
 REMOTE_SHIM_PORT = 10998
 REMOTE_RUNNER_PORT = 10999
+REQUEST_TIMEOUT = 5
```

Comment:
I recommend increasing it to 15 seconds.
```suggestion
REQUEST_TIMEOUT = 15
```